src/main/java/frc/team2412/robot/subsystem/DrivebaseSubsystem.java

```diff
@@ -94,17 +105,23 @@
     private final NetworkTableEntry odometryXEntry;
     private final NetworkTableEntry odometryYEntry;
     private final NetworkTableEntry odometryAngleEntry;
+    private final NetworkTableEntry module1, module2, module3, module4;
     private final NetworkTableEntry isFieldOrientedEntry;
 
-    public DrivebaseSubsystem(SwerveModule fl, SwerveModule fr, SwerveModule bl, SwerveModule br, Gyroscope g) {
+    private final Field2d field = new Field2d();
+
+    public DrivebaseSubsystem(SwerveModule fl, SwerveModule fr, SwerveModule bl, SwerveModule br, Gyroscope g,
+            double moduleMaxVelocityMetersPerSec) {
         synchronized (sensorLock) {
             gyroscope = g;
             gyroscope.setInverted(false);
+            SmartDashboard.putData("Field", field);
         }
```

Comment:
Robot.java already did this, so likely can be deleted here.

Reply:
No it doesnt work then